imports/plugins/included/payments-payflow-pro/client/templates/settings/payflow.html

```diff
@@ -9,15 +9,15 @@
   {{/unless}}
   {{#autoForm
     collection=Collections.Packages
-    schema=PaypalPackageConfig
+    schema=PayflowProPackageConfig
     doc=packageData
     type="update"
-    id="paypal-update-form-payflow"
+    id="payflow-update-form"
     }}
     {{>afQuickField name='settings.client_id'}}
     {{>afQuickField name='settings.client_secret' type="password"}}
-    {{>afQuickField name="settings.payflow.support" options="allowed" noselect=true}}
-    {{>afQuickField name='settings.payflow_mode' type="boolean-select" trueLabel="Live - Production Mode" falseLabel="Testing - Sandbox Mode"}}
+    {{>afQuickField name="settings.reaction-payflow-pro.support" options="allowed" noselect=true}}
```

Comment:
These really need to be refactored to a react settings component. https://github.com/reactioncommerce/reaction/issues/2198